python/pyspark/testing/pandasutils.py

```diff
@@ -65,6 +65,12 @@ def setUpClass(cls):
         super(PandasOnSparkTestCase, cls).setUpClass()
         cls.spark.conf.set(SPARK_CONF_ARROW_ENABLED, True)
 
+    def _convert_str_to_lambda(self, func):
```

Comment:
Let's remove leading underscore since this file isn't documented, packaged together, and test-only.
```suggestion
    def convert_str_to_lambda(self, func):
```